backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/AndroidApplicationBase.java

```diff
@@ -91,4 +88,8 @@
 	/** Returns the Handler object created by the application
 	 * @return The {@link Handler} object created by the application */
 	Handler getHandler ();
+
+	/** Returns the current Android Activity
+	 * @return The current Android {@link Activity} */
+	Activity getActivity ();
```

Comment:
You probably don't need this, use `getContext()` and check if it returns instance of `Activity`

Reply:
I don't think ``Context`` supports ``isInMultiWindowMode()``?

Reply:
`Activity` is a subclass of `Context` and in standard Android apps getContext() returns an Activity being LiveWallpapers an exception. I'm suggesting to check if `getContext()`returns an instance of `Activity` to then be able to call `isInMultiWindowMode()`.

Reply:
I see your point, there indeed is a ``getContext()`` method. I agree with obigu, let's remove the ``getActivity()`` and use the ``getContext()``! :)